packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -1,33 +1,64 @@
-import { act, render, screen } from "@testing-library/react"
-import React from "react"
+import { act, fireEvent, render, screen, waitFor } from "@testing-library/react"
+import React, { useState } from "react"
+import { FieldMessageStatus } from "@kaizen/draft-form"
 import userEvent from "@testing-library/user-event"
-import { DatePicker } from "./DatePicker"
+import { DatePicker, ValidationResponse } from "./DatePicker"
 import "@testing-library/jest-dom"
+import { DatePickerProps } from "."
 
 const defaultProps = {
   id: "date-picker",
   labelText: "Choose date",
   selectedDay: undefined,
   initialMonth: new Date(2022, 2),
   onDayChange: jest.fn(),
+  onValidate: jest.fn(),
+}
+
+const MockDatePicker = ({
```

Comment:
Maybe `DataPickerHarness` instead of `MockDatePicker`? There’s no mocking here?

Reply:
Will review the naming. It's more common practice to just call these wrappers though I believe (at least, in most examples of tests I've seen).